src/function_wrappers.ts

```diff
@@ -122,6 +126,25 @@ const wrapHttpFunction = (execute: HttpFunction): RequestHandler => {
   };
 };
 
+const wrapOpenFunction = (
+  userFunction: OpenFunction,
+  context: OpenFunctionContext
+): RequestHandler => {
+  const ctx = OpenFunctionRuntime.ProxyContext(context);
+
+  const httpHandler = (req: Request, res: Response) => {
+    const callback = getOnDoneCallback(res);
+    ctx.setTrigger(req, res);
+
+    Promise.resolve()
+      .then(() => userFunction(ctx, req.body))
```

Comment:
Sorry for my late response, basically the new signature refactoring looks good.
One more thing, I think go ff's handling of function output and return error is more reasonable.
What if send fails? Here're more refs:
- automatically set http ret code based on function error code https://github.com/OpenFunction/functions-framework-go/blob/main/runtime/knative/knative.go#L66
- encapsule function output into context: https://github.com/OpenFunction/functions-framework-go/blob/f1e311b3bf253b00c3cf494826f57acd22224016/runtime/runtime.go#L134
- return error in function when send fails or other errors occur
-